src/components/tab-group/daikin-tab-group.ts

```
@@ -0,0 +1,223 @@
+import { LitElement, html, css, unsafeCSS } from "lit";
+import { customElement, property } from "lit/decorators.js";
+
+import tailwindStyles from "../../tailwind.css";
+import type { DaikinTab } from "../tab/daikin-tab";
+
+/**
+ * Tab Group that automatically manages child tabs.
+ *
+ * NOTE: If there is no tab available in the child elements, i.e., zero tabs, or if all tabs are disabled, it is ILLEGAL.
```

Should we throw an error for these edge cases?

---

I'm not sure how best to inform users (developers) when a component is used in a way that it is not intended to be used.
For example, frameworks such as React only show errors during development and are silent in production builds.
Should we incorporate such a mechanism?

---

lit-html seems to use `development` field of `exports` in [package.json](https://github.com/lit/lit/blob/main/packages/lit-html/package.json#L15-L28).
ref: https://nodejs.org/api/packages.html#community-conditions-definitions

Ultimately, I think it would be best to provide both development and production code, as lit-html does, but I'm not sure at what point we should do that.
As for building, it is easy to achieve as it is just a `vite build` twice with different modes.

---

@yodas7 Since one goal of a design system is to have uniform design/UI across projects, I think it's okay to throw the error in both development and production. 

If we aren't strict in the beginning, it would be extremely hard to change later. 
If there is a use case which isn't covered, the user can file an issue/ticket.

---

@hiro-daikin 
Shortly after I wrote my comment, I changed it to include development builds.
Sorry for not notifying you.
I would appreciate it if you could check to see if this change looks good.